app.js

```diff
@@ -1,78 +1,78 @@
-// book class and constructor
+// book class - represents a book
 class Book {
   constructor(id, title, author) {
     this.id = id;
     this.title = title;
     this.author = author;
   }
-}
-
-// get books from localstorage
-const getBooks = () => {
-  let books;
 
-  if (localStorage.getItem('books') === null) {
-    books = [];
-  } else {
-    books = JSON.parse(localStorage.getItem('books'));
-  }
+  // get books from localstorage
+  static getBooks = () => {
```

Comment:
- [OPTIONAL] This method can be refactored like this 👇🏼 to make it more precise:
```JavaScript
static getBooks = () => {
    let books = JSON.parse(localStorage.getItem('books')) || [];
    return books;
};
```

Reply:
Thaks for the support!